components/sessions/FilterSessions.tsx

```diff
@@ -0,0 +1,264 @@
+                    name="session4"
+                    value={sessions[2]}
+                    onClick={(e) => onClickFilter(e)}
+                    className={`${
+                      filter?.session4 === sessions[2] && selectedClass
+                    } border rounded-r-md border-1 border-accent px-3 py-1 text-xs flex-1 text-center hover:cursor-pointer hover:bg-accent hover:text-black`}
+                  >
+                    {sessions[2]}
+                  </button>
+                )}
+              </div>
+            ))}
+          </div>
+
+          <div className="mt-10">
+            {groupBy3(sessions5).map((sessions: string[]) => (
+              <div
+                className={`flex pb-1 ${
+                  (sessions.length < 3 &&
+                    sessions.length % 3 === 1 &&
+                    'w-1/3') ||
+                  (sessions.length % 3 === 2 && 'w-2/3')
+                }`}
+              >
+                <button
+                  name="session5"
+                  type="button"
+                  value={sessions[0]}
+                  onClick={(e) => onClickFilter(e)}
+                  className={`${
+                    filter?.session5 === sessions[0] && selectedClass
+                  } border rounded-l-md border-1 border-accent px-3 py-1 text-xs flex-1 text-center hover:cursor-pointer hover:bg-accent hover:text-black`}
+                >
+                  {sessions[0]}
+                </button>
+                {sessions[1] && (
+                  <button
+                    name="session5"
+                    type="button"
+                    value={sessions[1]}
+                    onClick={(e) => onClickFilter(e)}
+                    className={`${
+                      filter?.session5 === sessions[1] && selectedClass
+                    } border border-1 border-accent px-3 py-1 text-xs flex-1 text-center hover:cursor-pointer hover:bg-accent hover:text-black`}
+                  >
+                    {sessions[1]}
+                  </button>
+                )}
+                {sessions[2] && (
+                  <button
+                    type="button"
+                    name="session5"
+                    value={sessions[2]}
+                    onClick={(e) => onClickFilter(e)}
+                    className={`${
+                      filter?.session5 === sessions[2] && selectedClass
+                    } border rounded-r-md border-1 border-accent px-3 py-1 text-xs flex-1 text-center hover:cursor-pointer hover:bg-accent hover:text-black`}
+                  >
+                    {sessions[2]}
+                  </button>
+                )}
+              </div>
+            ))}
+          </div>
+
+          <div className="mt-10">
+            <div className="flex pb-1">
+              <button
+                className="w-full rounded-md py-2 bg-primary font-bold"
+                type="button"
+              >
+                FILTER
```

Comment:
change to lowercase

Reply:
also there's a styling fo button
btn-primary

`className="btn-primary w-full"`